teuthology/suite/run.py

```diff
@@ -27,6 +27,45 @@
 
 log = logging.getLogger(__name__)
 
+def descr_to_yamls(descriptions, suites_path):
+    """
+    Function converts description of a job into sequence of .yaml files.
+    Input "descriptions" is a string containing comma-separated list of job descriptions.
+    Input "suites_path" is a posix path to dir containing all suites.
+    """
```

Comment:
That's way better, thank you, but could you please take a look at nice example of docstring usage https://github.com/ceph/teuthology/blob/main/teuthology/orchestra/remote.py#L221-L237